changelog.md

```diff
@@ -4,6 +4,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 # [Unreleased](https://github.com/MyPureCloud/genesys-cloud-webrtc-sdk/compare/v6.0.1...HEAD)
+
+### Fixed
```

Comment:
Move this section below PCM-1669. PCM-1669 isn't a bug fix its a new feature and belongs under "Added".

Reply:
Oh sorry about that! That was a merge conflict resolution and I didn't realize that happened